pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/painless/UpsertPainless.java

```diff
@@ -39,8 +40,13 @@ public UpsertPainless(Generator generator, String index, ScriptBuilder scriptBui
 
     @Override
     public void export() {
+        export(scriptBuilder.build(), generator.generate(), generator.getPropertyName());
+    }
+
+    public void export(String script, Map<String, Object> source, String propertyName) {
```

Comment:
I'm a little scared for this change because (please correct me if I'm wrong) but I think `export` is used a lot all over the code. Do we have enough testing coverage to know they are all going to be fine?

Reply:
Correct `export()` is used a lot and is unchanged. The in new `export(...)` is used in just one place.
However, I think this change is safe, and is somewhat self evident since `scriptBuilder` and `generator` are used only once in the prior version of this method, as you can see immediately below.

If it would make you more comfortable, I can make this method static by passing in the index.
I can also make a new constructor for this that does not include `Generator` or `ScriptBuilder`.